lib/components/map/default-map.tsx

```diff
@@ -264,6 +274,9 @@ class DefaultMap extends Component {
       lat: null,
       lon: null
     })
+
+    // Fetch feeds in the background
+    this.props.findFeeds()
```

Comment:
We fetch feeds every time the query changes? Is there no better time? If we do it like this can we at least cache the result

Reply:
doesn't componentDidMount just run once?